.github/workflows/tests.yml

```diff
@@ -46,6 +46,19 @@ jobs:
 
       - run: brew typecheck
 
+      - name: Check RuboCop filepaths
+        working-directory: Library/Homebrew
```

Comment:
This doesn't exist? https://github.com/Homebrew/brew/actions/runs/4293152654/jobs/7480482506#step:8:19

Reply:
🤔 It existed with the previous `actions/checkout` step, I wonder what `setup-homebrew` is doing?

Reply:
@issyl0 Maybe just running in another directory?

Reply:
Fixed!